pkg/virt-controller/watch/application.go

```diff
@@ -534,7 +532,10 @@ func (vca *VirtControllerApp) onStartedLeading() func(ctx context.Context) {
 			metrics.UpdateVMIMigrationInformer(vca.migrationInformer)
 		}
 		golog.Printf("\nvca.migrationInformer :%v\n", vca.migrationInformer)
-		migration.RegisterMigrationMetrics(vca.migrationInformer)
+
+		if err := metrics.AddVMIMigrationPhaseTransitionTimeFromCreationHandler(vca.migrationInformer); err != nil {
```

Comment:
it is not a direct match from `RegisterMigrationMetrics,` but this replaces the old `newVMIMigrationPhaseTransitionTimeFromCreationHistogramVec` (therefore the name) method that registered migration handlers. The old method in case of error would just `panic`.

the naming `CreateVMIMigrationHandler` could be a good fit